empress/recon_vis/tree.py

```diff
@@ -19,17 +24,52 @@ def __init__(self, name):
         self.parent_node = None     # Node:  parent Node or None
         self.layout = None          # NodeLayout object: layout of this node
 
+    # The @property decorator allows this to be called as .is_leaf rather than .is_leaf()
+    @property
     def is_leaf(self):
         """ returns True iff this node is a leaf/tip of the tree """
         return self.left_node is None and self.right_node is None
 
+    # The @property decorator allows this to be called as .is_root rather than .is_root()
+    @property
     def is_root(self):
         """ returns True iff this node is the root of the tree """
         return self.parent_node is None
 
     def __repr__(self):
         return str(self.name)
 
+    def get_layout(self):
+        """ returns the four values listed in NodeLayout"""
+        layout = self.layout
+        return layout.row, layout.col, layout.x, layout.y
+
+    def set_layout(self, row=None, col=None, x=None, y=None):
+        """Sets the layout"""
+        layout = self.layout
+        layout.row = row if row != None else layout.row
+        layout.col = col if col != None else layout.col
+        layout.x = x if x != None else layout.x
+        layout.y = y if y != None else layout.y
+
+    def iter_track(self, track):
```

Comment:
Please rename this function to be indicative of what it does.

Reply:
Fixed